doc-content/drools-docs/src/main/asciidoc/DecisionEngine/con-traits.adoc

```diff
@@ -0,0 +1,96 @@
+[id='con-traits_{context}']
+
+= Traits in the {DECISION_ENGINE}
+Traits in {DECISION_ENGINE} allow you to model multiple dynamic types that do not fit in a class hierarchy. A trait is an interface that you can apply (and eventually remove) to an individual object at runtime. However, traits are not supported in the executable rule model, therefore traits functionality and its specific implementation code is moved to an individual module named `drools-traits`. To use traits, you can import the `drools-traits` module to Maven.
+
+The `drools-traits` module is based on the `drools-core` and `drools-compiler` module. The classes from `drools-core` and `drools-compiler` are moved to the `drools-traits` module, containing `.traits` prefix in package name and `-Impl` suffix in class name to avoid the split package problem.
```

Comment:
This was a technical detail to provide other engineers some details on the work done, but it's irrelevant to the end users. We can safely remove this paragraph and the next note

Reply:
I agree that this paragraph is not relevant for users and can be removed.